src/odemis/acq/align/spot.py

```diff
@@ -357,31 +358,46 @@ def FindGridSpots(image, repetition):
     Returns
     -------
     spot_coordinates : array like
-        A 2D array of shape (N, 2) containing the coordinates of the spots.
+        A 2D array of shape (N, 2) containing the coordinates of the spots,
+        in respect to the top left of the image.
     translation : tuple of two floats
+        Translation from the origin to the center of the grid in image space,
+        origin is top left of the image. Primary axis points down and the
+        secondary axis points to the right.
     scaling : tuple of two floats
+        Scaling factors for primary and secondary axis.
     rotation : float
+        Rotation in image space, positive rotation is clockwise.
     shear : float
+        Horizontal shear factor. A positive shear factor transforms a coordinate
+        in the positive x direction parallel to the x axis.
 
     """
+    # Find the center coordinates of the spots in the image.
     spot_positions = MaximaFind(image, repetition[0] * repetition[1])
     if len(spot_positions) < repetition[0] * repetition[1]:
         logging.warning('Not enough spots found, returning only the found spots.')
         return spot_positions, None, None, None
-    # Estimate transformation
+    # Estimate the two most common directions in the grid of spots, these form the transformation matrix.
     lattice_constants = EstimateLatticeConstant(spot_positions)
+    # Transpose to put the primary axis on the first row instead of the first column. In a transformation matrix, the
+    # first row should represent the transform for the primary axis, second row for the secondary, etc..
```

Comment:
That is another way of saying it as in line 388?
`        # If the determinant is smaller than zero flip the transformation matrix to make it a right handed
        # transformation matrix.`
If yes maybe you could add here that is a RH transformation matrix then and in line 388 you could add the description with the primary and secondary axis.